### DISCUSSION TITLE
希望恢复首页的我的动态一栏

### DISCUSSION THREAD

**Discussion by JiaRuiShao** [the1812/Bilibili-Evolved#2844] [功能建议 / Ideas]
更新之后的版本（2.0+）找不到之前的版本（1.0+）首页里的动态栏了，求求把这个选项加到现在的版本插件里

**Comment by Geeyun-JY3** [comment]
v2 的清爽首页还没做完：https://github.com/the1812/Bilibili-Evolved/discussions/2717#discussioncomment-1843239
请认真看更新日志：#2640
